Add mortality chart

petrbouchal opened the pull request:
Ahoj, konečně jsem se k tomu dobral.

Je to docela kopa kódu a dat, protože je tam potřeba převést věkové skupiny a na to není nic off the shelf - čili jsem přidal skript na stažení těch mezivstupů, které se nemění co den/týden. 

Pokusil jsem se co nejvíc sladit styly, ale kdyžtak klidně upravuj, ideálně jako edit k pull requestu (https://docs.github.com/en/free-pro-team@latest/github/collaborating-with-issues-and-pull-requests/committing-changes-to-a-pull-request-branch-created-from-a-fork).

Metodologicky je to tak, že úmrtí v každé věkové skupině jsou počítané na 100 000 lidí v té věkové skupině. V roce 2020 používám data za 2019. Šedý pás v grafu je rozpětí takto normalizovaných úmrtností v letech 2011-19. 

Alternativa by byla zobrazit čáry za jednotlivé roky nebo nějaký střed (asi za méně let zpět), ale toto mi přišlo přehlednější a analyticky snad OK.

![image](https://user-images.githubusercontent.com/1666657/98486602-8e7f4200-221e-11eb-96d4-ad1dbabae499.png)

petrbouchal committed:
add mortality chart

petrbouchal committed:
exclude .zip from site to minimise footprint

petrbouchal committed:
translate label for 85+ age group

petrbouchal commented:
Tak ještě případně zadrž - u toho součtu (Total) je potřeba ten adjustment na věkovou strukturu udělat jinak. Pošlu ještě commit s opravou.

janxmares commented:
Ahoj Petře,

díky za doplnění. Dnes odpoledne jsem to otevřel. Počkám tedy na úpravy a
poté to projdu a spojím.

Honza

On Wed, 11 Nov 2020 at 18:51, Petr Bouchal <notifications@github.com> wrote:

> Tak ještě případně zadrž - u toho součtu (Total) je potřeba ten adjustment
> na věkovou strukturu udělat jinak, plus tam vidím chybu v podtitulku. Pošlu
> ještě commit s opravou.
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/janxmares/macrodash/pull/2#issuecomment-725567659>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ABYHW2KP7WHRF4OQJD2AQ43SPLFIHANCNFSM4TOTVLUQ>
> .
>


-- 
*Jan Mareš*
+420 739 411 839
janxmares@gmail.com

petrbouchal committed:
improve age-structure-adjustment

Previously, the "Total" category was not calculated as sum of deaths in each category adjusted for the size of that category, but as its own category, hence the age-normalised values in Total did not reflect the changes in age composition inside the Total from year to year. This is now fixed.

petrbouchal committed:
fix margin cutting off axis text

petrbouchal committed:
typo

petrbouchal commented:
Hotovo - jsou to nakonec absolutní čísla (představitelnější než per 100k), ale očištěná o vliv věkové proměny populace.

![image](https://user-images.githubusercontent.com/1666657/98852225-a5f53f80-2457-11eb-8bb3-70f4c25d6214.png)

janxmares commented on index.Rmd:
```diff
@@ -580,6 +582,131 @@ fig_morat
+       title = "Mortality (all-cause)",
+       subtitle = "Deaths per week, 2011-2020. Blue = 2020, grey = range in 2011-2019",
+       caption = str_glue("Based on CZSO dataset 130185 (deaths) and 130142 (age) | last checked {today_fmt} | Last data for {last_date_fmt}.")) +
+  geom_text(data = lbs_grey, aes(x = day_sameyear_floored, 
```
Petře, tyhle labely jsou na ose x, tj. daleko od samotných linií (koukám na graf z posledního komentáře). Nenašel jsem, že bys to v daších commits upravoval. Původně se zobrazovaly správně, ale změna normalizace to asi rozhodila. Myslím, že je potřeba upravit y. Nebo úplně vynechat?

petrbouchal replied:
Díky, to jsem úplně přehlídl. Byly to popisky přímo u dat, ale zapomněl jsem upravit souřadnice, když jsem to přehazoval na absolutní čísla. Je to takhle čitelné?

![image](https://user-images.githubusercontent.com/1666657/98971733-d7324600-2511-11eb-9396-e4a199127358.png)

janxmares replied:
Ta šedá je na hraně. Zkusil bych tmavší, zhruba jako popisky os?

petrbouchal replied:
ta v těch range pruzích?

janxmares replied:
Ano, jak je světlá na tmavé. Alespoň u mě se to čte hůře. Range bych nechal a dal bych tmavší popisek.

janxmares replied:
Koukám, že to navrhuješ obráceně. To je taky ok, jde jen o to mít tam větší kontrast.

petrbouchal replied:
Zkusil jsem to ještě otočit a je to myslím lepší...

janxmares replied:
Bezva, díky moc za doplnění! Jdu to mergnout.

janxmares reviewed (commented):
Styl grafů mi přijde dobrý!

petrbouchal committed:
clean up code: move all library calls into setup chunk

petrbouchal committed:
move direct labels

petrbouchal committed:
darker range ribbon bands

petrbouchal committed:
friendlier label contrast
